### ISSUE TITLE
New Round -> Can Save Higher Temperature than on Profile

### ISSUE THREAD

**Issue by saratheneale** [projectbuendia/client#362]
App version: (the number after "Buendia" on the starting screen)
Screen or dialog: New Round

#### Steps to replicate the problem

1. Add a New Patient Round
2. Enter 96 in Temperature field
3. Click Save

#### Actual behaviour
- the form saves 
#### Expected behaviour
To throw an error for the invalid temperature data. 

#### Notes 
The profile on staging.buendia's [openMRS Profile](https://docs.google.com/spreadsheets/d/13fmOhr64x4S2kNh7urgoViIAYCPqUr5xvccvNJEiXn8/edit#gid=725169659) is `drc-6`. The absolute range of temperature is 30-50.  I thought all of these limits were for form validation. I'm not sure what the expected behavior is here.

**Comment by zestyping** [comment]
Yes, this does look like a real bug.  The form should reject temperatures outside the range 30 to 50.

**Comment by zestyping** [comment]
Replicated this easily.
